tx/textual/internal/testpb/1.proto

```diff
@@ -55,3 +55,34 @@ message Bar {
   string bar_id = 1;
   bytes  data   = 2;
 }
+
+// Baz is a sample message type used for testing repeated rendering.
+message Baz {
+  repeated Foo messages   = 1;
+  repeated string string_messages = 2;
+  Vote vote = 3;
```

Comment:
Can we also test repeated Coins? it has a special rendering, just want to make sure it till works.

```suggestion
  repeated cosmos.base.v1beta1.Coin price = 8;
  Vote vote = 3;
```

Reply:
Done in 22f9b6c